linbox/matrix/matrixdomain/blas-matrix-domain.inl

```diff
@@ -32,2029 +32,353 @@
 #ifndef __LINBOX_matrix_matrixdomain_blas_matrix_domain_INL
 #define __LINBOX_matrix_matrixdomain_blas_matrix_domain_INL
 
-namespace LinBox { namespace Protected {
+#include "linbox/matrix/matrixdomain/blas-matrix-domain-mul.inl"
```

Comment:
File is missing.
Also in Makefile.am

Reply:
pushed right now, sorry.

Reply:
I am not sure about the static field in BlasPermutation, I don't have any clue but it might make things harder after hand.

Reply:
OK. Removed the static.